src/writeData/subscriptions/components/CreateSubscriptionPage.tsx

```diff
@@ -88,10 +91,10 @@ const CreateSubscriptionPage: FC = () => {
   useEffect(() => {
     event(
       'visited creation page',
-      {userAccountType: quartzMe?.accountType ?? 'unknown'},
+      {userAccountType: identity.account.type || 'unknown'},
```

Comment:
Due to differences in the reducers we need to use logical OR here. The nullish coalescing (??) operator was used because `quartzMe` had no default reducer state, so the fallback was triggered when `quartzMe` was undefined. 

quartzIdentity has a legit reducer state, and the default value for account.type is `''`. So now, we need the fallback to happen when `account.type` is falsy in any form, not `undefined`.

Reply:
>quartzIdentity has a legit reducer state, and the default value for account.type is `''`

It looks like it's [`free`](https://github.com/influxdata/ui/blob/master/src/identity/reducers/index.ts#L37). Seems like the whole fallback isn't necessary? Also, I think we shouldn't have different fallbacks in different parts of the app. In the above component, the fallback was `'free'` but here it's `'unknown'`. That seems like it has the potential to be really confusing to both people maintaining the codebase and to users

Reply:
Yes, you're right - and I think this can be resolved by your other comment. Since `free` is the default, sending this in the event name is just going to create a mismatch between the eventing data and what's in state.